idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/proofs/folding/tuples.kt

```diff
@@ -23,20 +25,27 @@ fun KtTypeReference.tupleTypeMatches(): Boolean =
     }
 
 private fun KotlinType?.isTypeMatching() =
```

Comment:
nit:
Maybe it's worth to set a parameter here with the FqName we want to target.

Reply:
thanks @i-walker ! updated! 👍